frontend/src/scenes/exports/ExportsList.tsx

```diff
@@ -62,11 +65,70 @@ export function Exports(): JSX.Element {
                             return <>{export_.destination.type}</>
                         },
                     },
+                    {
+                        title: 'Status',
+                        key: 'status',
+                        render: function RenderStatus(_, export_) {
+                            return (
+                                <>
+                                    {export_.paused === true ? (
+                                        <LemonTag type="default" className="uppercase">
+                                            Paused
+                                        </LemonTag>
+                                    ) : (
+                                        <LemonTag type="primary" className="uppercase">
+                                            Running
+                                        </LemonTag>
+                                    )}
+                                </>
+                            )
+                        },
+                    },
                     {
                         title: 'Frequency',
                         key: 'frequency',
                         dataIndex: 'interval',
                     },
+                    {
+                        title: 'Actions',
+                        render: function Render(_, export_) {
+                            const {
+                                executeExportAction: pauseExport,
+                                loading: pausing,
+                                error: pauseError,
+                            } = useExportAction(currentTeamId, export_.id, 'pause')
+                            const {
+                                executeExportAction: resumeExport,
+                                loading: resuming,
+                                error: resumeError,
+                            } = useExportAction(currentTeamId, export_.id, 'unpause')
+
+                            return (
+                                <div className={clsx('flex flex-wrap')}>
+                                    <LemonButton
+                                        status="primary"
+                                        type="secondary"
+                                        onClick={() => {
+                                            export_.paused
+                                                ? resumeExport().then(() => {
+                                                      if (resumeError === null) {
+                                                          export_.paused = false
```

Comment:
It's interesting that the component updates here to rerender. I'm not actually sure how that is happening 🤔

Reply:
Ah I guess the update to the state from `useExportAction` causes a rerender ✅